debian/copyright

```diff
@@ -209,9 +209,9 @@ License: GPL-2+
  The media files of Widelands are distributed under the same license
  than the game itself. The blender source files are not included in the same
  source tree because of their size but can be retrieved online from
- https://bugs.launchpad.net/widelands-media/
+ https://launchpad.net/widelands-media/
```

Comment:
Uhm, did we not transfer this to github?

Reply:
No. We left it on Launchpad for now because of the big file sizes.